index.js

```diff
@@ -14,9 +14,56 @@ const manageRecipes = async () => {
     // Connection to the database "recipe-app"
     const dbConnection = await mongoose.connect(MONGODB_URI);
     console.log(`Connected to the database: "${dbConnection.connection.name}"`);
-
     // Before adding any recipes to the database, let's remove all existing ones
     await Recipe.deleteMany();
+    //Iteration 2
+    // Create a recipe
+    
+
+    const newRecipes = await Recipe.create({
+      title: "Orange and Milk-Braised Pork Carnitas",
+      level: "UltraPro Chef",
+      ingredients: [
+        "3 1/2 pounds boneless pork shoulder, cut into large pieces",
+        "1 tablespoon freshly ground black pepper",
+        "1 tablespoon kosher salt, or more to taste",
+        "2 tablespoons vegetable oil",
+        "2 bay leaves",
+        "2 teaspoons ground cumin",
+        "1 teaspoon dried oregano",
+        "1/4 teaspoon cayenne pepper",
+        "1 orange, juiced and zested",
+      ],
+      cuisine: "American",
+      dishType: "main_course",
+      image:
+        "https://images.media-allrecipes.com/userphotos/720x405/2280918.jpg",
+      duration: 160,
+      creator: "Chef John",
+    });
+    console.log("Recipe created: ", newRecipes);
+
+    //Iteration 3
+    //Create many recipes
+    const manyRecipes = await Recipe.insertMany(data);
+    console.log("Created new Recipes:", manyRecipes);
```

Comment:
If you wanted to `console.log` only the titles of the recipes, you needed to loop over your `allRecipes` variable that is storing an array with your recipes objects.
Something like this:
```
// console log to see the titles of the inserted recipes
    for (let i = 0; i < manyRecipes.length; i++) {
      console.log(manyRecipes[i].title);
    }
```